src/features/bookings/components/TicketBody/DefaultBody/DefaultBody.tsx

```diff
@@ -0,0 +1,64 @@
+import { t } from '@lingui/macro'
+import React, { FunctionComponent } from 'react'
+import styled from 'styled-components/native'
+
+import { WithdrawalTypeEnum } from 'api/gen'
+import {
+  getDelayMessage,
+  getStartMessage,
+} from 'features/bookings/components/TicketBody/ticketBodyMessages'
+import { BicolorEmailSent as InitialBicolorEmailSent } from 'ui/svg/icons/BicolorEmailSent'
+import { getSpacing, Typo } from 'ui/theme'
+
+type Props = {
+  withdrawalType: WithdrawalTypeEnum.by_email | WithdrawalTypeEnum.on_site
+  withdrawalDelay: number
+}
+
+export const DefaultBody: FunctionComponent<Props> = ({ withdrawalType, withdrawalDelay }) => {
```

Comment:
je propose `TicketWithdrawal`
```suggestion
export const TicketWithdrawal: FunctionComponent<Props> = ({ withdrawalType, withdrawalDelay }) => {
```